### PR TITLE
Added HT#7 implementation

### PR DESCRIPTION
**Pull request by volkorezova**

### CODE REVIEW

File: homeworks/tatyana.volkorezova_volkorezova/7-class/baseTask.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+class CarSize {
+    constructor(height, width, carLength) {
+        this.height = height;
+        this.width = width;
+        this.carLength = carLength;
+    }
+}
+
+for (let i = 1; i <= WHEEL_AMOUNT; i++) {
```
Comment (mykola-yarovenko-moc) [inline homeworks/tatyana.volkorezova_volkorezova/7-class/baseTask.js:36]: image
empty item?

File: homeworks/tatyana.volkorezova_volkorezova/7-class/baseTask.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    wheelsPackForVolvo[i] = new CarWheel('16 duyms', '200 mm');
```
Comment (mykola-yarovenko-moc) [inline homeworks/tatyana.volkorezova_volkorezova/7-class/baseTask.js:37]: image
image

upper case in object key?

File: homeworks/tatyana.volkorezova_volkorezova/7-class/baseTask.js (hunk 1 above)
Comment (volkorezova) [reply homeworks/tatyana.volkorezova_volkorezova/7-class/baseTask.js]: fixed

File: homeworks/tatyana.volkorezova_volkorezova/7-class/baseTask.js (hunk 2 above)
Comment (volkorezova) [reply homeworks/tatyana.volkorezova_volkorezova/7-class/baseTask.js:37]: renamed keys

**Review by mykola-yarovenko-moc — APPROVED** [review]

**Pull request merged**